packages/aws-cdk-lib/aws-eks/lib/managed-nodegroup.ts

```diff
@@ -597,6 +597,7 @@ function isGpuInstanceType(instanceType: InstanceType): boolean {
   //compare instanceType to known GPU InstanceTypes
   const knownGpuInstanceTypes = [InstanceClass.P2, InstanceClass.P3, InstanceClass.P3DN, InstanceClass.P4DE, InstanceClass.P4D,
     InstanceClass.G3S, InstanceClass.G3, InstanceClass.G4DN, InstanceClass.G4AD, InstanceClass.G5, InstanceClass.G5G,
+    InstanceClass.G6, InstanceClass.GR6,
     InstanceClass.INF1, InstanceClass.INF2];
```

Comment:
```suggestion
    InstanceClass.G6, InstanceClass.GR6, InstanceClass.INF1, InstanceClass.INF2];
```
It should fit in one line.